libs/eo/shared/atomic-design/ui-atoms/src/lib/eo-beta-message/eo-beta-message.component.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { WattButtonComponent } from '@energinet-datahub/watt/button';
+import { WattCardComponent } from '@energinet-datahub/watt/card';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  selector: 'eo-eo-beta-message',
+  standalone: true,
+  imports: [CommonModule, WattButtonComponent, WattCardComponent],
+  templateUrl: './eo-beta-message.component.html',
+  styleUrls: ['./eo-beta-message.component.scss'],
+})
+export class EoBetaMessageComponent {
+  isVisible = true;
+
+  closeMessage() {
+    this.isVisible = false;
+  }
```

Comment:
This can be deleted

Reply:
deleted in [7e9915a](https://github.com/Energinet-DataHub/greenforce-frontend/pull/1967/commits/7e9915a94f42c0fb737c41582fc6f34e4ec9dbbb)